### PR TITLE
Hotfix/corner sides fix

### PR DESCRIPTION
**Pull request by mmadjer**
This is a hot fix for the newly created corner sides support where the corner sides will not render properly.

### CODE REVIEW

**Review by tbaranes — APPROVED** [review]
All good, feel free to merge it 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ None
 None
 
 #### Bugfixes
-None
+- Fix initial `cornerSide` rendering issue.
```
Comment (JakeLin) [inline CHANGELOG.md:15]: We always put the issue or PR there for reference. like `[#367](https://github.com/IBAnimatable/IBAnimatable/pull/367)`

**Review by JakeLin — APPROVED** [review]
Thanks for fixing it.

**Comment by JakeLin** [comment]
@mmadjer Thanks for fixing this issue quickly. Sorry to get back to you so late, I am still on parental leave. Can you merge it when you free it is ready? You can merge it once you get a 👍  (Approve).

**Pull request merged**